Source/Data/05 - DefaultSettings.sql

```diff
@@ -856,4 +856,20 @@ INSERT INTO CellCarrier (Name,Transform) VALUES
 ('Cricket Wireless', '{0}@mms.cricketwireless.net'),
 ('Republic Wireless','{0}@text.republicwireless.com'),
 ('Google Fi','{0}@msg.fi.google.com')
+GO
+
+INSERT INTO [MiMD.DBCleanUpTask] (SQLCommand, Schedule, Name) VALUES
```

Comment:
`05 - Default Settings.sql` runs before `10 - MiMD.sql` so the table doesn't exist yet.

Reply:
Moved it, thank you for the catch